src/calculateRentalCost.js

```diff
@@ -5,6 +5,17 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const basePrice = days * 40;
```

Comment:
Consider defining a constant for the daily rental cost instead of using the literal value `40`. This will make the code more understandable and easier to maintain if the daily cost changes in the future.